File: 백준/Silver/1764. 듣보잡/듣보잡.swift
import Foundation

let nm = readLine()!.split { $0 == " " }.map { Int($0)! }
var setA = Set<String>()
var setB = Set<String>()

for _ in 0..<nm[0] {
  setA.insert(readLine()!)
}

for _ in 0..<nm[1] {
  setB.insert(readLine()!)
}

var result = setA.intersection(setB).sorted()
print(result.count)
result.forEach {
  print($0)
}

File: 백준/Silver/1764. 듣보잡/듣보잡.py
import sys

readline = sys.stdin.readline

setA = set()
setB = set()

n, m = map(int, readline().split())
setA = set([readline().rstrip() for _ in range(n)])
setB = set([readline().rstrip() for _ in range(m)])

setR = setA.intersection(setB)
result = sorted(setR)
print(len(result))
for name in result:
    print(name)
